packages/shared_preferences/shared_preferences_macos/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: shared_preferences_macos
 description: macOS implementation of the shared_preferences plugin.
-version: 0.0.1+3
+version: 0.0.2
```

Comment:
Should be 0.0.1+4 as this is a bug fix (update the version in the change log as well)